src/vmm/src/vstate.rs

```diff
@@ -717,7 +720,6 @@ impl Vcpu {
     pub fn new_x86_64(
         id: u8,
         vm_fd: &VmFd,
-        cpuid: CpuId,
```

Comment:
`cpuid` should be removed from the list of arguments in the function comment. Also, the comment on line 731 should be updated/removed.

Reply:
done.